ansible/deploy.yml

```diff
@@ -13,9 +21,14 @@
         requirements: /home/ubuntu/flask-alb-app/requirements.txt
         extra_args: "--no-cache-dir"
 
-    - name: Include vars file
-      include_vars:
-        file: vars.yml
+    - name: Copy local script to server
+      ansible.builtin.copy:
+          src: ../packer/scripts/user_data/app.sh
+          dest: /home/ubuntu/script.sh
+          mode: '0755'
+
+    - name: Run script on server
+      command: /home/ubuntu/script.sh
```

Comment:
Btw, You can add a kind of idempotent behavior here with `creates:` option
[link to Ansible docs](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/command_module.html#parameter-creates)
example:
```
 - name: Run script on server
      command: /home/ubuntu/script.sh
      creates: /home/ubuntu/app.env
```
in this case if file specified in `creates:` option exists module will be skipped.
It's not necessary here, but can be useful in future.

Reply:
Cool, didn't know about that